.github/workflows/tests.yml

```
@@ -0,0 +1,53 @@
+name: Tests
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+    branches: [ master ]
```

```suggestion
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```

Might make sense to do this also for the `check-yourls-release` workflow, but less important.

I found in other projects using GHA that pull requests would receive CI results twice if the `push` event wasn't filtered. The workflow would run on _both_ the `push` and the PR `opened` or `synchronize` events, which pollutes the status box and (more importantly) wastes runner time.

Yeah, I prefer to be nice to GHA when possible, since it's a free service for OSS projects. 😸